tutorials/hist/ContourList.C

```diff
@@ -115,23 +116,23 @@ TCanvas *ContourList(){
    2, -2, 2, 2, 0, 6.5);
 
    hr->Draw();
-   Double_t x0, y0, z0;
+   Double_t xval0, yval0, zval0;
```

Comment:
It was needed, but only because the script that generates the notebooks removes the code from the main function and executes it outside the function. This can be seen when comparing [approx.C](https://root.cern.ch/doc/master/approx_8C_source.html) and its corresponding [notebook](http://nbviewer.jupyter.org/url/root.cern.ch/doc/master/notebooks/approx.C.nbconvert.ipynb). It should work if the code is executed inside a function.